.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
         "editor.formatOnSave": true,
         "editor.defaultFormatter": "charliermarsh.ruff",
         "editor.codeActionsOnSave": {
-            "source.organizeImports": "explicit"
+            "source.organizeImports": "never"
```

Comment:
I think @fbdtemme doesn't use vscode, so these settings shouldn't matter to him. I do sort them, but only manually, because when the setting above is not "never", there is some bug in the vscode plugin that makes it remove code. Which is rather annoying...